src/components/table/useTableController.js

```diff
@@ -40,7 +40,11 @@ const useTableController = (options) => {
     if (params) {
       Object.entries(params).forEach(([key, value]) => {
         if (value) {
-          urlParams.set(key, value.toString());
+          if (Array.isArray(value)) {
+            value.forEach((val) => {
+              urlParams.append(key, val);
+            });
```

Comment:
this code doesn't work

Reply:
it depends https://github.com/3v-workspace/Data_converter/pull/1169

Reply:
nope, its doesn't work anyway...